packages/fx-core/src/common/samples-config.json

```diff
@@ -50,6 +50,20 @@
             "shortDescription": "Search and Share an NPM Package in Teams and Outlook (Web)",
             "fullDescription": "NPM Search Connector is a Messaging Extension that runs across M365 including Teams and Outlook (Web). It allows you to perform a quick search for a NPM package and insert package details into conversations for easy sharing. The front end is built with Adaptive Cards and the backend is an Azure Bot Service.",
             "tags": ["Message Extension", "JS", "Outlook"]
+        },
+        {
+            "id": "hello-world-tab",
```

Comment:
Is it expected to put the hello world app to the last of all sample apps instead of the 1st place?

Reply:
Updated, thanks